python/test/gluon/test_frontend.py

```diff
@@ -1429,3 +1429,53 @@ def test_infer_layout_for_amd_mfma(target):
   }
 }
 """)
+
+
+@gluon.jit
+def buffer_ldst_kernel(x, y):
+    layout: ttgl.constexpr = ttgl.BlockedLayout(size_per_thread=[1, 1], threads_per_warp=[1, 64], warps_per_cta=[4, 1],
+                                                order=[1, 0])
+
+    input_offsets = ttgl.arange(0, 64 * 64).reshape(64, 64)
+    input_offsets = ttgl.convert_layout(input_offsets, layout)
+    mask = ttgl.full((64, 64), 1, tl.int1, layout=layout)
+    a = ttgl.amd.cdna3.buffer_load(ptr=x, offsets=input_offsets)
+
+    output_offsets = ttgl.arange(0, 64 * 64).reshape(64, 64)
+    output_offsets = ttgl.convert_layout(output_offsets, layout)
+    ttgl.amd.cdna3.buffer_store(stored_value=a, ptr=y, offsets=output_offsets, mask=mask, cache='.ca')
+
+    other = ttgl.full((64, 64), 1.0, tl.float32, layout=layout)
+    a = ttgl.amd.cdna3.buffer_load(ptr=x, offsets=input_offsets, mask=mask, other=other)
+    ttgl.amd.cdna3.buffer_store(stored_value=a, ptr=y, offsets=output_offsets, mask=mask, cache='.ca')
```

Comment:
You may want to add `cache` to `buffer_load` here too. And the test from L1439 - L1446 doesn't provide much more value given this is a superset of that; you can remove prior ones.